ImageViewer/Source/GalleryPagingDataSource.swift

```diff
@@ -19,14 +19,15 @@ final class GalleryPagingDataSource: NSObject, UIPageViewControllerDataSource {
     fileprivate var itemCount: Int { return itemsDataSource?.itemCount() ?? 0 }
     fileprivate unowned var scrubber: VideoScrubber
 
-    init(itemsDataSource: GalleryItemsDataSource, displacedViewsDataSource: GalleryDisplacedViewsDataSource?, scrubber: VideoScrubber, configuration: GalleryConfiguration) {
+    init(itemsDataSource: GalleryItemsDataSource?, displacedViewsDataSource: GalleryDisplacedViewsDataSource?, scrubber: VideoScrubber, configuration: GalleryConfiguration) {
```

Comment:
I am not too sure about this optional at the point of initialisation. Can you tell me in what scenario do you envision this happening?